quickwit-indexing/src/models/indexed_split.rs

```diff
@@ -86,6 +86,7 @@ impl IndexedSplit {
             uncompressed_docs_size_in_bytes: 0,
             time_range: None,
             replaced_split_ids: Vec::new(),
+            delete_opstamp: 0, // TODO: fetch the last delete task opstamp to set it.
```

Comment:
is it something we can either fix before merge or open an issue?

Reply:
We can yes, this is not a hard pb, I'm just annoyed by the fact that I will probably need to add the metastore to the `IndexerState` in order to fetch the task.